appservice/src/index.ts

```diff
@@ -19,7 +19,7 @@ export * from './deleteSite';
 export * from './deploy/deploy';
 export * from './deploy/getDeployFsPath';
 export * from './deploy/IDeployContext';
-export * from './deploy/runPreDeployTask';
+export { handleFailedPreDeployTask, IPreDeployTaskResult, runPreDeployTask, tryRunPreDeployTask } from './deploy/runDeployTask';
```

Comment:
Why are you exporting `handleFailedPreDeployTask`?  I don't see it used in https://github.com/microsoft/vscode-azurefunctions/pull/1996/files

Reply:
It's used here: https://github.com/microsoft/vscode-azurefunctions/blob/master/src/commands/deploy/runPreDeployTask.ts#L35

All the functions listed here already existed and were exported previously. The only thing _not_ exported out of `runDeployTask.ts` is `startPostDeployTask` which is the function I just added

Reply:
Oh doh, you were * exporting.  Sorry 😬

Reply:
![dontworry](https://media.giphy.com/media/26gryOqhfD7cbKVS8/giphy.gif)